guides/v2.1/comp-mgr/upgrader/ce-ee-upgrade-start.md

```diff
@@ -11,40 +11,38 @@ functional_areas:
   - Upgrade
 ---
```

Comment:
Please complete these changes in v2.2 version if needed, as it is not symlinked.